lib/auth/webauthn/session.go

```diff
@@ -39,21 +39,40 @@ const scopeLogin = "login"
 // that use in-memory storage.
 const scopeSession = "registration"
 
-func sessionToPB(sd *wan.SessionData) (*wanpb.SessionData, error) {
+// SessionData is a clone of [protocol.SessionData], materialized here to keep a
+// stable JSON marshal/unmarshal representation and add extensions.
+type SessionData struct {
+	// Raw challenge used for the ceremony.
+	Challenge []byte `json:"challenge,omitempty"`
+	// Raw User ID.
+	UserId []byte `json:"userId,omitempty"`
+	// Raw Credential IDs of the credentials allowed for the ceremony.
+	AllowCredentials [][]byte `json:"allowCredentials,omitempty"`
+	// True if resident keys were required by the server / Relying Party.
+	ResidentKey bool `json:"residentKey,omitempty"`
+	// Requested user verification requirement, either "discouraged" or
+	// "required".
+	// An empty value is treated equivalently to "discouraged".
+	UserVerification string `json:"userVerification,omitempty"`
+	// Extensions are Teleport extensions that apply to this webauthn session.
+	Extensions *proto.MFAChallengeExtensions `json:"extensions,omitempty"`
```

Comment:
`Extensions` already has [meaning in WebAuthn][1] and it something we could add in the future. Let's call this something more specific, `ChallengeExtensions` maybe?

[1]: https://pkg.go.dev/github.com/go-webauthn/webauthn/webauthn#SessionData